### ISSUE TITLE
The Plugin doesnt work in 1.19.4 - Please add support for 1.19.4[BUG] 

### ISSUE THREAD

**Issue by CloudeLecaw** [PluginBugs/Issues-RPGBank#15]
**Describe the bug**
The plugin is not loading

**To Reproduce**
1 Start  Paper Server with latest 1.19.4 and latest LonLibs and Latest RPGBank Version

**Expected behavior**
The Plugin should work for 1.19.4

**Errorcode**
```
[02:27:58] [Server thread/INFO]: [RPGBank] Enabling RPGBank v1.0.16
[02:27:58] [Server thread/INFO]: 
[02:27:58] [Server thread/INFO]: -=[ RPGBank ]=-
[02:27:58] [Server thread/INFO]: 
[02:27:58] [Server thread/INFO]: Version: 1.0.16
[02:27:58] [Server thread/INFO]: Server version: git-Paper-453 (MC: 1.19.4)
[02:27:58] [Server thread/INFO]: 
[02:27:58] [Server thread/INFO]: 
[02:27:58] [Server thread/INFO]: [RPGBank] Detected Vault!
[02:27:58] [Server thread/ERROR]: [RPGBank] Failed to hook PlaceholderAPI! Make sure you have the latest version and it's not a corrupted JAR.
[02:27:58] [Server thread/WARN]: java.lang.AbstractMethodError: Missing implementation of resolved method 'abstract java.lang.String getIdentifier()' of abstract class me.clip.placeholderapi.expansion.PlaceholderExpansion.
[02:27:58] [Server thread/WARN]: 	at PlaceholderAPI_2.11.3-Build160.jar//me.clip.placeholderapi.expansion.manager.LocalExpansionManager.register(LocalExpansionManager.java:214)
[02:27:58] [Server thread/WARN]: 	at PlaceholderAPI_2.11.3-Build160.jar//me.clip.placeholderapi.expansion.PlaceholderExpansion.register(PlaceholderExpansion.java:139)
[02:27:58] [Server thread/WARN]: 	at RPGBank_1.0.16.jar//dev.lone.RPGBank.c.a.e(SourceFile:7)
[02:27:58] [Server thread/WARN]: 	at RPGBank_1.0.16.jar//dev.lone.RPGBank.Main.onEnable(SourceFile:83)
[02:27:58] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:279)
[02:27:58] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:192)
[02:27:58] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
[02:27:58] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
[02:27:58] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:555)
[02:27:58] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:466)
[02:27:58] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:637)
[02:27:58] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:436)
[02:27:58] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
[02:27:58] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1103)
[02:27:58] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
[02:27:58] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by LoneDev6** [comment]
Will be fixed, no plugin was currently updated to 1.19.4.

**Comment by CloudeLecaw** [comment]
looks like it doesnt hook with latest PlaceholderAPI :\```
[19:21:20] [Server thread/ERROR]: [RPGBank] Failed to hook PlaceholderAPI! Make sure you have the latest version and it's not a corrupted JAR.
[19:21:20] [Server thread/WARN]: java.lang.AbstractMethodError: Missing implementation of resolved method 'abstract java.lang.String getIdentifier()' of abstract class me.clip.placeholderapi.expansion.PlaceholderExpansion.
[19:21:20] [Server thread/WARN]: 	at PlaceholderAPI_2.11.3-Build173.jar//me.clip.placeholderapi.expansion.manager.LocalExpansionManager.register(LocalExpansionManager.java:214)
[19:21:20] [Server thread/WARN]: 	at PlaceholderAPI_2.11.3-Build173.jar//me.clip.placeholderapi.expansion.PlaceholderExpansion.register(PlaceholderExpansion.java:139)
[19:21:20] [Server thread/WARN]: 	at _RPGBank_1.0.16.jar//dev.lone.RPGBank.c.a.e(SourceFile:7)
[19:21:20] [Server thread/WARN]: 	at _RPGBank_1.0.16.jar//dev.lone.RPGBank.Main.onEnable(SourceFile:83)
[19:21:20] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:279)
[19:21:20] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:192)
[19:21:20] [Server thread/WARN]: 	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
[19:21:20] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
[19:21:20] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:555)
[19:21:20] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:466)
[19:21:20] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:637)
[19:21:20] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:436)
[19:21:20] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
[19:21:20] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1103)
[19:21:20] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
[19:21:20] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```
